src/when-you-get-an-alert.html

```diff
@@ -72,7 +72,10 @@ <h2 class="govuk-heading-m">
         If you want to see an alert again
       </h2>
       <p class="govuk-body">
-        You can find current and past alerts on your phone or tablet.
+        Visit gov.uk/alerts to see <a href="/alerts/current-alerts">current alerts</a> and <a href="/alerts/current-alerts">past alerts</a>.
```

Comment:
Does it sound weird to tell users to visit a site they’re already on?

It’s less active, but might be better to say:

```suggestion
        You can find <a href="/alerts/current-alerts">current alerts</a> and <a href="/alerts/current-alerts">past alerts</a> at gov.uk/alerts.
```

Reply:
When I read this I did find it a bit weird. I prefer your suggestion

Reply:
Thanks @idavidmcdonald.

Normally we wouldn’t mention the URL at all – but this content is trying to do 2 things:

* signpost the current and past alerts pages
* teach users what to do in the future